xbmc/settings/AdvancedSettings.h

```diff
@@ -312,6 +313,8 @@ class CAdvancedSettings : public ISettingCallback, public ISettingsHandler
     int m_iPVRNumericChannelSwitchTimeout; /*!< @brief time in msecs after that a channel switch occurs after entering a channel number, if confirmchannelswitch is disabled */
     int m_iPVRTimeshiftThreshold; /*!< @brief time diff between current playing time and timeshift buffer end, in seconds, before a playing stream is displayed as timeshifting. */
     bool m_bPVRTimeshiftSimpleOSD; /*!< @brief use simple timeshift OSD (with progress only for the playing event instead of progress for the whole ts buffer). */
+    CViewState m_PVRDefaultViewState; /*!< @brief CViewState used to store default description sort type and sort order */
```

Comment:
No need for a 'CViewState' as you are only using the 'SortDescription'.

Reply:
NP.  The original settings test file is still ok, but now for the wiki  it is 

```
  <pvr>
    <pvrrecordings>
      <sortmethod>2</sortmethod> <!-- 1=Name 2=Date 3=Size 4=File -->
      <sortorder>2</sortorder>  <!-- 1=Ascending 2=Descending -->
    </pvrrecordings>
  </pvr>
```